src/components/LocationSelect.tsx

```diff
@@ -0,0 +1,29 @@
+import type { Location } from '../types/sparkeats';
+
+type LocationSelectProps = {
+  locations: Location[];
+};
+
+function LocationSelect({ locations }: LocationSelectProps) {
+  const allCities = locations.map((location) => location.city);
+  const uniqueCities = [...new Set(allCities)];
+
+  return (
+    <div className="location-select">
+      <h2 className="location-select__title">Pick a Location</h2>
```

Comment:
I scanned the page with[ axe DevTools](https://www.deque.com/axe/devtools/) and found one critical issue.

<img width="895" alt="Screenshot 2022-10-30 at 12 25 15 PM" src="https://user-images.githubusercontent.com/16216104/198861375-d5e6ce83-6431-417d-8c2a-4ef41906ec46.png">

I think we need to replace this with a `label` and update the styles if necessary.

Reply:
Updated in f5f7392d531c852a4cb3adf073b6fdffbee05b3a.